opencti-platform/opencti-graphql/src/modules/notification/notification-domain.ts

```diff
@@ -65,11 +79,16 @@ export const triggerDelete = async (context: AuthContext, user: AuthUser, trigge
   return triggerId;
 };
 export const triggersFind = (context: AuthContext, user: AuthUser, opts: QueryTriggersArgs) => {
-  return listEntitiesPaginated<BasicStoreEntityTrigger>(context, user, [ENTITY_TYPE_TRIGGER], opts);
-};
-
-export const myTriggersFind = (context: AuthContext, user: AuthUser, opts: QueryTriggersArgs) => {
-  const queryFilters = [...(opts.filters || []), { key: 'user_ids', values: [user.id] }];
+  let queryFilters;
+  const hasSetAccessCapability = isUserHasCapability(user, SETTINGS_SET_ACCESSES);
+  if (opts.recipientId && opts.recipientId !== user.id) {
```

Comment:
Same as above, recipients_id should be a filter and not a api parameter. And I don't think we should enforce user_ids here, let's talk about it